boa/src/builtins/number/mod.rs

```diff
@@ -354,7 +354,7 @@ impl Number {
                 fraction -= digit as f64;
                 // Round to even.
                 if fraction + delta > 1.0
-                    && (fraction > 0.5 || (fraction - 0.5) < f64::EPSILON && digit & 1 != 0)
+                    && (fraction > 0.5 || (fraction - 0.5).abs() < f64::EPSILON && digit & 1 != 0)
```

Comment:
For my understanding why is this .abs()? Won't this change the behaviour of the code?

Reply:
Yeah it will but if we’re comparing to epsilon typically you do want Abs rather than the flat value, I couldn’t figure out why it wasn’t Abs in the first place.